src/js/ui/editor/menu.js

```diff
@@ -80,6 +80,24 @@ export function renderSelectColorPosition(ctrl) {
         )
       ])
     ]),
+    m('div.select_input', [
+      m('label', {
+        'for': 'select_editor_positions'
+      }, 'Endgames'),
+      m('select.positions', {
+        id: 'select_editor_positions',
+        onchange: function(e) {
+          ctrl.loadNewFen(e.target.value);
+        }
+      }, [
+        optgroup('Set the board', [
```

Comment:
All changes done ! 
I have a question, how do you launch the project in browser ? Last week is worked well, 
but now after 'npm run watch', i have to do '  python -m SimpleHTTPServer 8080' 
If not, anything happen in browser. 
Even after that, it work, but i still have s cors issue and i'm not able to make any request